### ISSUE TITLE
Insufficient directory permissions

### ISSUE THREAD

**Issue by raupie** [ezsystems/LegacyBridge#73]
I walked through the install procedure and when I access /ezsetup I'm getting an Issue that I'm missing some directories. I edited the config.php file in `ezpublish_legacy` folder for changing the ezpublish folder to app. I tried clearing the cache. 

```
eZ Publish cannot create some important directories, without this the setup cannot finish and parts of eZ Publish will fail.
The nonexistent directories are: ../ezpublish/logs ../ezpublish/cache ../ezpublish/config

You can try the following shell commands to create necessary directories:

cd /var/www/restek.com/public_html/ezpublish_legacy
mkdir -p ../ezpublish/logs \
../ezpublish/cache \
../ezpublish/config

```

**Comment by emodric** [comment]
Hm... I can't remember having these issues when I last ran the setup wizard. I'll try to test it again though.

**Comment by raupie** [comment]
Just tried going through the installation again in a different VM. Same issue. 

I can see in the LegacySetupController.php file that the CheckList array contains the /ezpublish/logs... hardcoded.

**Comment by emodric** [comment]
@raupie Wanna do a PR to fix it?

**Comment by raupie** [comment]
@emodric, The controller was last modified a year ago, I can't believe I'm the first person to notice. I changed the checklist paths from ezpublish to app and that removed the warning message and I could proceed through the installation. I don't know enough about ezpublish to do a PR as I think maybe I did something else wrong. I'm not sure though unless someone can duplicate what I'm seeing.

**Comment by emodric** [comment]
> The controller was last modified a year ago, I can't believe I'm the first person to notice.

Well, nobody runs the setup wizard any more I guess :)

> I don't know enough about ezpublish to do a PR as I think maybe I did something else wrong. I'm not sure though unless someone can duplicate what I'm seeing.

Just do a PR to this repo with your changes, we'll take care of it and let you know if it's ok or not :)

**Comment by andrerom** [comment]
Fixed in #74
